### PR TITLE
This model the Trayectory

### PR DESCRIPTION
**Pull request by Eljuan25**
In this section we create the trajectory model, with its respective many-to-many or one-to-many models. but services is not available maybe that's why it causes a small error.

### CODE REVIEW

**Comment by Eljuan25** [comment]
I missed it in the services section, I put it in plural instead of singular sorry

**Comment by Eljuan25** [comment]
modify some service changes and accommodate it

File: app/models/trajectory.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class Trajectory < ApplicationRecord
+    has_and_belongs_to_many :stops
+    belongs_to :city
+    belongs_to :service
+
+    validates :city_id, presence: true 
+    validates :services_id, presence: true
```
Comment (mtopete1996) [inline app/models/trajectory.rb]: This is already validated by the belongs_to, so remove these lines.

**Review by mtopete1996 — CHANGES_REQUESTED** [review]

**Review by mtopete1996 — APPROVED** [review]
Ok looks good now

File: test/models/trayecctoy_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class TrayecctoyTest < ActiveSupport::TestCase
```
Comment (jesusMava) [inline test/models/trayecctoy_test.rb]: which one is the correct TrayecctoyTest or TrajectoryTest?

**Review by jesusMava — CHANGES_REQUESTED** [review]

File: test/models/trayecctoy_test.rb (hunk 2 above)
Comment (Eljuan25) [reply test/models/trayecctoy_test.rb]: ya its delete trayecctoy

**Review by jesusMava — APPROVED** [review]

**Pull request merged**